media/starboard/starboard_cdm.cc

```diff
@@ -367,10 +367,9 @@ void StarboardCdm::OnSessionUpdated(int ticket,
   } else {
     promises_.RejectPromise(session_update.promise_id,
                             CdmPromise::Exception::INVALID_STATE_ERROR, 0,
-                            "Unable to create CDM.");
+                            error_message);
```

Comment:
The original Cobalt implementation is to guard the case that CloseSession() is called before the OnSessionUpdateRequestGenerated() is called. The best way to verify this is to create a test web page that generates a update request then immediately close it, add an extra delay in OnSessionUpdateRequestGenerated() callback, so it called after OnSessionClosed(), and see if this is handled correctly.